src/vs/editor/contrib/find/browser/simpleFindWidget.contribution.ts

```diff
@@ -0,0 +1,151 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Command } from 'vs/editor/common/editorCommonExtensions';
+import { ServicesAccessor } from 'vs/platform/instantiation/common/instantiation';
+import { KeyCode, KeyMod } from 'vs/base/common/keyCodes';
+import { KeybindingsRegistry } from 'vs/platform/keybinding/common/keybindingsRegistry';
+import { ISimpleFindWidgetService, KEYBINDING_CONTEXT_SIMPLE_FIND_WIDGET_ACTIVE } from 'vs/editor/contrib/find/browser/simpleFindWidgetService';
+import { SimpleFindWidget, KEYBINDING_CONTEXT_SIMPLE_FIND_WIDGET_INPUT_FOCUSED } from 'vs/editor/contrib/find/browser/simpleFindWidget';
+
+export const SIMPLE_FIND_IDS = {
+	ShowWidgetSimpleFindCommand: 'editor.action.simplefind.show',
```

Comment:
@rebornix 
That's whythe comment musing about using 'workbench.XYZ'
isn't that what represents global commands?